app/routes.ts

```diff
@@ -6,4 +6,5 @@ export default [
   route("/second-opinion", "routes/second-opinion.tsx"),
   route("/clear-details", "routes/clear-details.tsx"),
   route("/delete-detail", "routes/delete-detail.tsx"),
+  route("/__vite_ping", "routes/noop.tsx")
```

Comment:
What's the purpose of this route? Would have assumed any kind of internal Vite healthcheck is handled by the Vite dev server itself.